xml/release-notes.xml

```diff
@@ -357,184 +353,9 @@
    <itemizedlist>
     <listitem>
      <para>
-      <package>artha</package>: Removed because it is unmaintained and has
-      unpatched security issues. See
+     <package>gap</package>: Removed because package is not FHS-compliant
       <link xlink:href="https://bugzilla.opensuse.org/show_bug.cgi?id=1143860"/>.
```

Comment:
```suggestion
     <package>gap</package>: Removed because the package is not FHS-compliant.
      For more information, see <link xlink:href="https://bugzilla.opensuse.org/show_bug.cgi?id=1143860"/>.
```